src/main/java/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -127,10 +127,10 @@ class HMPPSDomainEventsEmitter(
       .withAdditionalInformation("probableCause", receivedReason.probableCause?.name)
       .withAdditionalInformation("source", receivedReason.source.name)
       .withAdditionalInformation("details", receivedReason.details)
-      .withAdditionalInformation("currentLocation", receivedReason.currentLocation.name)
+      .withAdditionalInformation("currentLocation", receivedReason.currentLocation?.name)
```

Comment:
this is odd - the emitter would keel over in a heap previously if `currentLocation` is `null`, however Prison API Service returned `null` under some circumstances.  I've decided to go for caution and allow it still to be `null` and stop it all falling over if it is